app/bundles/EmailBundle/Entity/EmailRepository.php

```diff
@@ -29,11 +30,16 @@ class EmailRepository extends CommonRepository
     public function getDoNotEmailList()
     {
         $q = $this->_em->getConnection()->createQueryBuilder();
-        $q->select('lower(e.address) as email')
-            ->from(MAUTIC_TABLE_PREFIX.'email_donotemail', 'e');
+        $q->select('distinct(l.email)')
+            ->from(MAUTIC_TABLE_PREFIX . 'lead_donotcontact', 'dnc')
+            ->leftJoin('dnc', MAUTIC_TABLE_PREFIX . 'leads', 'l', 'l.id = dnc.lead_id')
+            ->where('dnc.channel = "email"')
+            ->where('l.email != ""');
+
         $results = $q->execute()->fetchAll();
 
         $dnc = array();
+
         foreach ($results as $r) {
             $dnc[] = $r['email'];
```

Comment:
We should either add back `lower()` in the select query or use `strtolower()` here since the comparison in `EmailModel::sendEmail()` is using ```if (in_array(strtolower($lead['email']), $dnc))```. In theory case should match but the given email in sendEmail() isn't necessarily guaranteed to be from the lead's table.  Or if we keep this as is, let's change that line in sendEmail to not force a lower case on the email.